activejob/lib/active_job/railtie.rb

```diff
@@ -34,7 +34,11 @@ class Railtie < Rails::Railtie # :nodoc:
 
         options.each do  |k, v|
           k = "#{k}="
-          send(k, v) if respond_to? k
+          if ActiveJob.respond_to?(k)
+            ActiveJob.send(k, v)
+          elsif respond_to? k
+            send(k, v)
+          end
```

Comment:
This does change the behaviour if the setter exists on **both `ActiveJob` and `ActiveJob::Base`, as it will now no longer be set on `Base`. We could alternatively do something like
```suggestion
          ActiveJob.send(k, v) if ActiveJob.respond_to? k
          send(k, v) if respond_to? k
```

Reply:
I should also note I wasn't able to find any settings for the Railtie. Are there any? If so, it may be worth updating them.

Reply:
Nah it's fine. It works like this in others gems line Active Record.